V1/auth/views.py

```diff
@@ -0,0 +1,156 @@
+
+from V1.models import User
+import uuid
+from flask_jwt_extended import (
+    JWTManager, jwt_required, create_access_token,
+    get_jwt_identity, get_raw_jwt
+)
+from werkzeug.security import check_password_hash
+from V1.auth import validate  # pragma: no cover
+from V1 import app  # pragma: no cover
+from flask import Blueprint,\
+    jsonify, request, make_response  # pragma: no cover
+
+auth_blueprint = Blueprint('auth', __name__)
+users = []
+logged_in_user = None
+
+# Setup the Flask-JWT-Extended extension
+app.config['JWT_SECRET_KEY'] = 'super-secret'
+app.config['JWT_BLACKLIST_ENABLED'] = True
+app.config['JWT_BLACKLIST_TOKEN_CHECS'] = ['access']
+jwt = JWTManager(app)
+blacklist = set()
+
+
+@jwt.token_in_blacklist_loader
+def check_if_token_in_blacklist(decrypted_token):
+    '''Black listing'''
```

Comment:
A good example of a comment within a function would be """Takes in a username and returns a boolean """